templates/cakephp/source.latte

```diff
@@ -15,6 +15,5 @@ the file LICENSE.md that was distributed with this source code.
 
 {block #content}
 {var $lineRegex = '~<span class="line">(\\s*(\\d+):\\s*)</span>([^\\n]*(?:\\n|$))~'}
-<pre class="numbers"><code>{$source|replaceRE:$lineRegex,'<span class="l"><a href="#$2">$1</a></span>'|noescape}</code></pre>
-<pre class="code"><code>{$source|replaceRE:$lineRegex,'<span id="$2" class="l">$3</span>'|noescape}</code></pre>
+    <pre><code>{$source|replaceRE:$lineRegex,'<span id="$2" class="l"><a class="l" href="#$2">$1</a>$3</span>'|noescape}</code></pre>
```

Comment:
Nitpick: Either this indentation be removed or line above should also be indented.